debian/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+if [[ -z $(dpkg --print-foreign-architectures | grep armhf) ]]; then
+	sudo dpkg --add-architecture armhf
+fi
+if [[ -z $(dpkg --print-foreign-architectures | grep arm64) ]]; then
+	sudo dpkg --add-architecture arm64
+fi
+
+if [[ $# == 0 || ! -z $(echo $@ | grep kubernetes-cni) ]]; then
+
+	for arch in amd64 arm arm64; do
```

Comment:
can you extract these loop params to a bash array and move to the top of file for clarity?